preprocessing/nextclade/src/loculus_preprocessing/prepro.py

```diff
@@ -201,40 +201,21 @@ def run_sort(
 
     for seq in missing_ids:
         alerts.warnings[seq].append(
-            ProcessingAnnotation(
-                unprocessedFields=[
-                    AnnotationSource(
-                        name="alignment", type=AnnotationSourceType.NUCLEOTIDE_SEQUENCE
-                    ),
-                ],
-                processedFields=[
-                    AnnotationSource(
-                        name="alignment", type=AnnotationSourceType.NUCLEOTIDE_SEQUENCE
-                    ),
-                ],
-                message=(
-                    "Sequence does not appear to match reference, per `nextclade sort`. "
-                    "Double check you are submitting to the correct organism."
-                ),
-            )
+            ProcessingAnnotation.from_single(
+                "alignment",
```

Comment:
This should probably be a constant - but we can make a new issue for that

Reply:
I can still try to add that in this PR as it isn't a real change :-)